ALCPlugFix-Swift/ALCPlugFix.swift

```diff
@@ -41,7 +46,19 @@ class ALCPlugFix {
     private func sendHDAVerb(_ command: HDAVerbModel) {
         // Otherwise, execute the commands
         print("Executing command labelled: \(command.comment ?? "No Description")")
-        runShellCommand(alcVerbPath, args: [command.nodeID, command.verb, command.param])
+        var connect: io_connect_t = 0;
```

Comment:
Lose the semicolon please.

Reply:
Oops, missed this one. Haven't got used to switching between C and Swift yet. XD